### PR TITLE
Fixing OCS API namespace usage

### LINKED ISSUE #15716: Class 'OCP\OC_API' not found in .../lib/public/api.php
```
Fatal error: Class 'OCP\OC_API' not found in /home/nickv/ownCloud/master/core/lib/public/api.php on line 51
Call Stack
#	Time	Memory	Function	Location
1	0.0007	230632	{main}( )	../index.php:0
2	0.0472	1534536	OC::handleRequest( )	../index.php:40
3	0.0702	2110352	OC\Route\Router->match( )	../base.php:870
4	0.0702	2110840	OC\Route\Router->loadRoutes( )	../router.php:243
5	0.0749	2269088	OC\Route\Router->requireRouteFile( )	../router.php:155
6	0.0753	2270672	include_once( '/home/nickv/ownCloud/master/repos/activity/appinfo/routes.php' )	../router.php:313
7	0.0755	2272888	OCP\API::register( )	../routes.php:34
```

This was broken since ever:
https://github.com/owncloud/core/pull/1053/files#diff-f4662541127f5c503384ba11c92b1d3fR40
All official apps use `\OC_API` at the moment, so I think fixing that in master or stable8 should be enough? @DeepDiver1975

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #15716

@DeepDiver1975 For stable8 I would only backport the first commit (Adding the missing slashes)

### CODE REVIEW

**Comment by nickvergessen** [comment]
> The inspection completed: 49 new issues

Added a commit that changes all usages of the deprecated constants

File: lib/public/api.php [hunk 1]
```diff
@@ -38,17 +38,33 @@
 class API {
 
 	/**
+	 * API authentication levels
```
Comment (MorrisJobke) [inline lib/public/api.php]: can you add a `@since 8.1.0`? Thanks

**Comment by karlitschek** [comment]
:+1:

**Comment by rullzer** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**